package.json

```diff
@@ -61,7 +61,7 @@
     "@typescript-eslint/eslint-plugin": "^5.9.0",
     "@typescript-eslint/parser": "^5.9.0",
     "concurrently": "^7.0.0",
-    "cypress": "^9.2.0",
+    "cypress": "^9.2.1",
```

Comment:
FWIW The ^ should mean you don't have to bump the semver manually

Reply:
yes, but my cypress was not starting up when I was trying out to test out my changes. Suggestions online were to bump up the cypress version. after doing that it was working ... which is strange